core/api/src/main/java/com/alipay/sofa/rpc/auth/AuthRuleGroup.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.auth;
+
+import java.util.List;
+
+/**
+ *
+ * @author lepdou
```

Comment:
Change author style like this:
https://github.com/sofastack/sofa-rpc/blob/f19e2d2757b1788010f190d47e449ed688b5d09c/core/api/src/main/java/com/alipay/sofa/rpc/api/GenericService.java#L22